### ISSUE TITLE
Image tag version based on helmfile version

### ISSUE THREAD

**Issue by agershman** [travisghansen/argo-cd-helmfile#64]
When using the built container image as an Argo CD repo server sidecar plugin, it feels like it would be more intuitive to have the image tag contain the version of helmfile in it (e.g. `0.167.1-1`). I do see that there are other things included in the `Dockerfile` which could be changed and affect functionality without having to change the version of helmfile itself. For this reason, I think it could be address by tacking a simple incrementing number to the helmfile version (`...-1` in the example). Though there are multiple pieces of software in the container, it's really helmfile that's the centerpiece. Curious if this has been considered or what the thoughts are around this?

**Comment by crabique** [comment]
I agree. Especially now that Helmfile `v1.0.0` with some breaking changes is about to be released, logically this would require a  major version bump for the plugin as well, and since it would bring it into v1.x.x territory it makes a lot of sense to just align the versions.

**Comment by crabique** [comment]
Now that [v1.0.0](https://github.com/helmfile/helmfile/releases/tag/v1.0.0) is released and it breaks some backwards compatibility, @travisghansen would it make sense if the versions for new releases would be matching the helmfile version?
